### ISSUE TITLE
[QUESTION] What is the difference between Olson Land_Map and Land_Type_Masks

### ISSUE THREAD

**Issue by xpye98** [geoschem/gchp#163]
I'm using the latest GCHP 13.2.1.

In `ExtData.rc`, we can see GCHP now reads land types from `./ChemDir/Olson_Land_Map_201203/Olson_2001_Land_Map.025x025.generic.nc`, 

but below there is another option to read land types from 
`./HcoDir/OLSON_MAP/v2019-02/Olson_2001_Land_Type_Masks.025x025.generic.nc`, which I believe is the default option in GC Classic.

My questions are,
1. Is there a difference between these two files, just from the perspective of data itself?
2. If no, for what reason does CCHP chooses the former one. Is it about regridding?

**Comment by LiamBindle** [comment]
That's a good question. It looks like the difference is that the `ChemDir/` version has a single variable that enumerates the land types whereas the `HcoDir/` version has 73 separate variables that are the fractional cover for each type.
 
<details>
<summary><code>$ ncdump -h ./ChemDir/Olson_Land_Map_201203/Olson_2001_Land_Map.025x025.generic.nc</code></summary>

```
netcdf Olson_2001_Land_Map.025x025.generic {
dimensions:
	lon = 1440 ;
	lat = 720 ;
	time = 1 ;
variables:
	float lon(lon) ;
		lon:long_name = "longitude" ;
		lon:units = "degrees_east" ;
	float lat(lat) ;
		lat:long_name = "latitude" ;
		lat:units = "degrees_north" ;
	int time(time) ;
		time:long_name = "time" ;
		time:units = "minutes since 1985-01-01 00:00:00 GMT" ;
		time:delta_t = "0000-00-00 00:00:00" ;
		time:begin_date = "19850101" ;
		time:begin_time = "000000" ;
		time:time_increment = "000000" ;
	int OLSON(time, lat, lon) ;
		OLSON:long_name = "Olson (2001) land map values" ;
		OLSON:units = "unitless" ;
		OLSON:gamap_category = "LANDMAP" ;
	float DXYP(time, lat, lon) ;
		DXYP:long_name = "Grid box surface area" ;
		DXYP:units = "m2" ;
		DXYP:gamap_category = "DXYP" ;

// global attributes:
		:title = "Olson (2001) Land Map data" ;
		:history = "23 Mar 2012" ;
		:conventions = "COARDS" ;
		:format = "netCDF-3" ;
		:model = "GENERIC" ;
		:nlayers = "1" ;
		:start_date = "19850101" ;
		:start_time = "00:00:00.0" ;
		:end_date = "19860101" ;
		:end_time = "23:59:59.0" ;
		:delta_lon = "0.25" ;
		:delta_lat = "0.25" ;
		:delta_time = "000000" ;
}

```

Here is a plot of OLSON
![image](https://user-images.githubusercontent.com/12772306/135935723-ebb7c9ae-1c3e-4e65-a371-07606b5d8932.png)

</details>


<details>
<summary><code>$ ncdump -h ./HcoDir/OLSON_MAP/v2019-02/Olson_2001_Land_Type_Masks.025x025.generic.nc</code></summary>

```
netcdf Olson_2001_Land_Type_Masks.025x025.generic {
dimensions:
	time = 1 ;
	lat = 720 ;
	lon = 1440 ;
variables:
	float LANDTYPE00(time, lat, lon) ;
		LANDTYPE00:_FillValue = 9.96921e+36f ;
		LANDTYPE00:long_name = "Land type mask" ;
		LANDTYPE00:units = "1" ;
		LANDTYPE00:_ChunkSizes = 1, 720, 1440 ;
	int time(time) ;
		time:long_name = "time" ;
		time:units = "minutes since 1985-01-01 00:00:00 GMT" ;
		time:delta_t = "0000-00-00 00:00:00" ;
		time:begin_date = "19850101" ;
		time:begin_time = "000000" ;
		time:time_increment = "000000" ;
	float lat(lat) ;
		lat:long_name = "latitude" ;
		lat:units = "degrees_north" ;
	float lon(lon) ;
		lon:long_name = "longitude" ;
		lon:units = "degrees_east" ;
	float LANDTYPE01(time, lat, lon) ;
		LANDTYPE01:_FillValue = 9.96921e+36f ;
		LANDTYPE01:long_name = "Land type mask" ;
		LANDTYPE01:units = "1" ;
		LANDTYPE01:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE02(time, lat, lon) ;
		LANDTYPE02:_FillValue = 9.96921e+36f ;
		LANDTYPE02:long_name = "Land type mask" ;
		LANDTYPE02:units = "1" ;
		LANDTYPE02:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE03(time, lat, lon) ;
		LANDTYPE03:_FillValue = 9.96921e+36f ;
		LANDTYPE03:long_name = "Land type mask" ;
		LANDTYPE03:units = "1" ;
		LANDTYPE03:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE04(time, lat, lon) ;
		LANDTYPE04:_FillValue = 9.96921e+36f ;
		LANDTYPE04:long_name = "Land type mask" ;
		LANDTYPE04:units = "1" ;
		LANDTYPE04:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE05(time, lat, lon) ;
		LANDTYPE05:_FillValue = 9.96921e+36f ;
		LANDTYPE05:long_name = "Land type mask" ;
		LANDTYPE05:units = "1" ;
		LANDTYPE05:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE06(time, lat, lon) ;
		LANDTYPE06:_FillValue = 9.96921e+36f ;
		LANDTYPE06:long_name = "Land type mask" ;
		LANDTYPE06:units = "1" ;
		LANDTYPE06:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE07(time, lat, lon) ;
		LANDTYPE07:_FillValue = 9.96921e+36f ;
		LANDTYPE07:long_name = "Land type mask" ;
		LANDTYPE07:units = "1" ;
		LANDTYPE07:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE08(time, lat, lon) ;
		LANDTYPE08:_FillValue = 9.96921e+36f ;
		LANDTYPE08:long_name = "Land type mask" ;
		LANDTYPE08:units = "1" ;
		LANDTYPE08:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE09(time, lat, lon) ;
		LANDTYPE09:_FillValue = 9.96921e+36f ;
		LANDTYPE09:long_name = "Land type mask" ;
		LANDTYPE09:units = "1" ;
		LANDTYPE09:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE10(time, lat, lon) ;
		LANDTYPE10:_FillValue = 9.96921e+36f ;
		LANDTYPE10:long_name = "Land type mask" ;
		LANDTYPE10:units = "1" ;
		LANDTYPE10:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE11(time, lat, lon) ;
		LANDTYPE11:_FillValue = 9.96921e+36f ;
		LANDTYPE11:long_name = "Land type mask" ;
		LANDTYPE11:units = "1" ;
		LANDTYPE11:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE12(time, lat, lon) ;
		LANDTYPE12:_FillValue = 9.96921e+36f ;
		LANDTYPE12:long_name = "Land type mask" ;
		LANDTYPE12:units = "1" ;
		LANDTYPE12:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE13(time, lat, lon) ;
		LANDTYPE13:_FillValue = 9.96921e+36f ;
		LANDTYPE13:long_name = "Land type mask" ;
		LANDTYPE13:units = "1" ;
		LANDTYPE13:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE14(time, lat, lon) ;
		LANDTYPE14:_FillValue = 9.96921e+36f ;
		LANDTYPE14:long_name = "Land type mask" ;
		LANDTYPE14:units = "1" ;
		LANDTYPE14:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE15(time, lat, lon) ;
		LANDTYPE15:_FillValue = 9.96921e+36f ;
		LANDTYPE15:long_name = "Land type mask" ;
		LANDTYPE15:units = "1" ;
		LANDTYPE15:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE16(time, lat, lon) ;
		LANDTYPE16:_FillValue = 9.96921e+36f ;
		LANDTYPE16:long_name = "Land type mask" ;
		LANDTYPE16:units = "1" ;
		LANDTYPE16:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE17(time, lat, lon) ;
		LANDTYPE17:_FillValue = 9.96921e+36f ;
		LANDTYPE17:long_name = "Land type mask" ;
		LANDTYPE17:units = "1" ;
		LANDTYPE17:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE18(time, lat, lon) ;
		LANDTYPE18:_FillValue = 9.96921e+36f ;
		LANDTYPE18:long_name = "Land type mask" ;
		LANDTYPE18:units = "1" ;
		LANDTYPE18:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE19(time, lat, lon) ;
		LANDTYPE19:_FillValue = 9.96921e+36f ;
		LANDTYPE19:long_name = "Land type mask" ;
		LANDTYPE19:units = "1" ;
		LANDTYPE19:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE20(time, lat, lon) ;
		LANDTYPE20:_FillValue = 9.96921e+36f ;
		LANDTYPE20:long_name = "Land type mask" ;
		LANDTYPE20:units = "1" ;
		LANDTYPE20:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE21(time, lat, lon) ;
		LANDTYPE21:_FillValue = 9.96921e+36f ;
		LANDTYPE21:long_name = "Land type mask" ;
		LANDTYPE21:units = "1" ;
		LANDTYPE21:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE22(time, lat, lon) ;
		LANDTYPE22:_FillValue = 9.96921e+36f ;
		LANDTYPE22:long_name = "Land type mask" ;
		LANDTYPE22:units = "1" ;
		LANDTYPE22:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE23(time, lat, lon) ;
		LANDTYPE23:_FillValue = 9.96921e+36f ;
		LANDTYPE23:long_name = "Land type mask" ;
		LANDTYPE23:units = "1" ;
		LANDTYPE23:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE24(time, lat, lon) ;
		LANDTYPE24:_FillValue = 9.96921e+36f ;
		LANDTYPE24:long_name = "Land type mask" ;
		LANDTYPE24:units = "1" ;
		LANDTYPE24:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE25(time, lat, lon) ;
		LANDTYPE25:_FillValue = 9.96921e+36f ;
		LANDTYPE25:long_name = "Land type mask" ;
		LANDTYPE25:units = "1" ;
		LANDTYPE25:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE26(time, lat, lon) ;
		LANDTYPE26:_FillValue = 9.96921e+36f ;
		LANDTYPE26:long_name = "Land type mask" ;
		LANDTYPE26:units = "1" ;
		LANDTYPE26:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE27(time, lat, lon) ;
		LANDTYPE27:_FillValue = 9.96921e+36f ;
		LANDTYPE27:long_name = "Land type mask" ;
		LANDTYPE27:units = "1" ;
		LANDTYPE27:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE28(time, lat, lon) ;
		LANDTYPE28:_FillValue = 9.96921e+36f ;
		LANDTYPE28:long_name = "Land type mask" ;
		LANDTYPE28:units = "1" ;
		LANDTYPE28:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE29(time, lat, lon) ;
		LANDTYPE29:_FillValue = 9.96921e+36f ;
		LANDTYPE29:long_name = "Land type mask" ;
		LANDTYPE29:units = "1" ;
		LANDTYPE29:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE30(time, lat, lon) ;
		LANDTYPE30:_FillValue = 9.96921e+36f ;
		LANDTYPE30:long_name = "Land type mask" ;
		LANDTYPE30:units = "1" ;
		LANDTYPE30:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE31(time, lat, lon) ;
		LANDTYPE31:_FillValue = 9.96921e+36f ;
		LANDTYPE31:long_name = "Land type mask" ;
		LANDTYPE31:units = "1" ;
		LANDTYPE31:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE32(time, lat, lon) ;
		LANDTYPE32:_FillValue = 9.96921e+36f ;
		LANDTYPE32:long_name = "Land type mask" ;
		LANDTYPE32:units = "1" ;
		LANDTYPE32:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE33(time, lat, lon) ;
		LANDTYPE33:_FillValue = 9.96921e+36f ;
		LANDTYPE33:long_name = "Land type mask" ;
		LANDTYPE33:units = "1" ;
		LANDTYPE33:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE34(time, lat, lon) ;
		LANDTYPE34:_FillValue = 9.96921e+36f ;
		LANDTYPE34:long_name = "Land type mask" ;
		LANDTYPE34:units = "1" ;
		LANDTYPE34:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE35(time, lat, lon) ;
		LANDTYPE35:_FillValue = 9.96921e+36f ;
		LANDTYPE35:long_name = "Land type mask" ;
		LANDTYPE35:units = "1" ;
		LANDTYPE35:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE36(time, lat, lon) ;
		LANDTYPE36:_FillValue = 9.96921e+36f ;
		LANDTYPE36:long_name = "Land type mask" ;
		LANDTYPE36:units = "1" ;
		LANDTYPE36:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE37(time, lat, lon) ;
		LANDTYPE37:_FillValue = 9.96921e+36f ;
		LANDTYPE37:long_name = "Land type mask" ;
		LANDTYPE37:units = "1" ;
		LANDTYPE37:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE38(time, lat, lon) ;
		LANDTYPE38:_FillValue = 9.96921e+36f ;
		LANDTYPE38:long_name = "Land type mask" ;
		LANDTYPE38:units = "1" ;
		LANDTYPE38:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE39(time, lat, lon) ;
		LANDTYPE39:_FillValue = 9.96921e+36f ;
		LANDTYPE39:long_name = "Land type mask" ;
		LANDTYPE39:units = "1" ;
		LANDTYPE39:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE40(time, lat, lon) ;
		LANDTYPE40:_FillValue = 9.96921e+36f ;
		LANDTYPE40:long_name = "Land type mask" ;
		LANDTYPE40:units = "1" ;
		LANDTYPE40:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE41(time, lat, lon) ;
		LANDTYPE41:_FillValue = 9.96921e+36f ;
		LANDTYPE41:long_name = "Land type mask" ;
		LANDTYPE41:units = "1" ;
		LANDTYPE41:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE42(time, lat, lon) ;
		LANDTYPE42:_FillValue = 9.96921e+36f ;
		LANDTYPE42:long_name = "Land type mask" ;
		LANDTYPE42:units = "1" ;
		LANDTYPE42:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE43(time, lat, lon) ;
		LANDTYPE43:_FillValue = 9.96921e+36f ;
		LANDTYPE43:long_name = "Land type mask" ;
		LANDTYPE43:units = "1" ;
		LANDTYPE43:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE44(time, lat, lon) ;
		LANDTYPE44:_FillValue = 9.96921e+36f ;
		LANDTYPE44:long_name = "Land type mask" ;
		LANDTYPE44:units = "1" ;
		LANDTYPE44:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE45(time, lat, lon) ;
		LANDTYPE45:_FillValue = 9.96921e+36f ;
		LANDTYPE45:long_name = "Land type mask" ;
		LANDTYPE45:units = "1" ;
		LANDTYPE45:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE46(time, lat, lon) ;
		LANDTYPE46:_FillValue = 9.96921e+36f ;
		LANDTYPE46:long_name = "Land type mask" ;
		LANDTYPE46:units = "1" ;
		LANDTYPE46:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE47(time, lat, lon) ;
		LANDTYPE47:_FillValue = 9.96921e+36f ;
		LANDTYPE47:long_name = "Land type mask" ;
		LANDTYPE47:units = "1" ;
		LANDTYPE47:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE48(time, lat, lon) ;
		LANDTYPE48:_FillValue = 9.96921e+36f ;
		LANDTYPE48:long_name = "Land type mask" ;
		LANDTYPE48:units = "1" ;
		LANDTYPE48:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE49(time, lat, lon) ;
		LANDTYPE49:_FillValue = 9.96921e+36f ;
		LANDTYPE49:long_name = "Land type mask" ;
		LANDTYPE49:units = "1" ;
		LANDTYPE49:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE50(time, lat, lon) ;
		LANDTYPE50:_FillValue = 9.96921e+36f ;
		LANDTYPE50:long_name = "Land type mask" ;
		LANDTYPE50:units = "1" ;
		LANDTYPE50:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE51(time, lat, lon) ;
		LANDTYPE51:_FillValue = 9.96921e+36f ;
		LANDTYPE51:long_name = "Land type mask" ;
		LANDTYPE51:units = "1" ;
		LANDTYPE51:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE52(time, lat, lon) ;
		LANDTYPE52:_FillValue = 9.96921e+36f ;
		LANDTYPE52:long_name = "Land type mask" ;
		LANDTYPE52:units = "1" ;
		LANDTYPE52:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE53(time, lat, lon) ;
		LANDTYPE53:_FillValue = 9.96921e+36f ;
		LANDTYPE53:long_name = "Land type mask" ;
		LANDTYPE53:units = "1" ;
		LANDTYPE53:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE54(time, lat, lon) ;
		LANDTYPE54:_FillValue = 9.96921e+36f ;
		LANDTYPE54:long_name = "Land type mask" ;
		LANDTYPE54:units = "1" ;
		LANDTYPE54:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE55(time, lat, lon) ;
		LANDTYPE55:_FillValue = 9.96921e+36f ;
		LANDTYPE55:long_name = "Land type mask" ;
		LANDTYPE55:units = "1" ;
		LANDTYPE55:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE56(time, lat, lon) ;
		LANDTYPE56:_FillValue = 9.96921e+36f ;
		LANDTYPE56:long_name = "Land type mask" ;
		LANDTYPE56:units = "1" ;
		LANDTYPE56:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE57(time, lat, lon) ;
		LANDTYPE57:_FillValue = 9.96921e+36f ;
		LANDTYPE57:long_name = "Land type mask" ;
		LANDTYPE57:units = "1" ;
		LANDTYPE57:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE58(time, lat, lon) ;
		LANDTYPE58:_FillValue = 9.96921e+36f ;
		LANDTYPE58:long_name = "Land type mask" ;
		LANDTYPE58:units = "1" ;
		LANDTYPE58:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE59(time, lat, lon) ;
		LANDTYPE59:_FillValue = 9.96921e+36f ;
		LANDTYPE59:long_name = "Land type mask" ;
		LANDTYPE59:units = "1" ;
		LANDTYPE59:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE60(time, lat, lon) ;
		LANDTYPE60:_FillValue = 9.96921e+36f ;
		LANDTYPE60:long_name = "Land type mask" ;
		LANDTYPE60:units = "1" ;
		LANDTYPE60:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE61(time, lat, lon) ;
		LANDTYPE61:_FillValue = 9.96921e+36f ;
		LANDTYPE61:long_name = "Land type mask" ;
		LANDTYPE61:units = "1" ;
		LANDTYPE61:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE62(time, lat, lon) ;
		LANDTYPE62:_FillValue = 9.96921e+36f ;
		LANDTYPE62:long_name = "Land type mask" ;
		LANDTYPE62:units = "1" ;
		LANDTYPE62:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE63(time, lat, lon) ;
		LANDTYPE63:_FillValue = 9.96921e+36f ;
		LANDTYPE63:long_name = "Land type mask" ;
		LANDTYPE63:units = "1" ;
		LANDTYPE63:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE64(time, lat, lon) ;
		LANDTYPE64:_FillValue = 9.96921e+36f ;
		LANDTYPE64:long_name = "Land type mask" ;
		LANDTYPE64:units = "1" ;
		LANDTYPE64:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE65(time, lat, lon) ;
		LANDTYPE65:_FillValue = 9.96921e+36f ;
		LANDTYPE65:long_name = "Land type mask" ;
		LANDTYPE65:units = "1" ;
		LANDTYPE65:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE66(time, lat, lon) ;
		LANDTYPE66:_FillValue = 9.96921e+36f ;
		LANDTYPE66:long_name = "Land type mask" ;
		LANDTYPE66:units = "1" ;
		LANDTYPE66:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE67(time, lat, lon) ;
		LANDTYPE67:_FillValue = 9.96921e+36f ;
		LANDTYPE67:long_name = "Land type mask" ;
		LANDTYPE67:units = "1" ;
		LANDTYPE67:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE68(time, lat, lon) ;
		LANDTYPE68:_FillValue = 9.96921e+36f ;
		LANDTYPE68:long_name = "Land type mask" ;
		LANDTYPE68:units = "1" ;
		LANDTYPE68:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE69(time, lat, lon) ;
		LANDTYPE69:_FillValue = 9.96921e+36f ;
		LANDTYPE69:long_name = "Land type mask" ;
		LANDTYPE69:units = "1" ;
		LANDTYPE69:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE70(time, lat, lon) ;
		LANDTYPE70:_FillValue = 9.96921e+36f ;
		LANDTYPE70:long_name = "Land type mask" ;
		LANDTYPE70:units = "1" ;
		LANDTYPE70:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE71(time, lat, lon) ;
		LANDTYPE71:_FillValue = 9.96921e+36f ;
		LANDTYPE71:long_name = "Land type mask" ;
		LANDTYPE71:units = "1" ;
		LANDTYPE71:_ChunkSizes = 1, 720, 1440 ;
	float LANDTYPE72(time, lat, lon) ;
		LANDTYPE72:_FillValue = 9.96921e+36f ;
		LANDTYPE72:long_name = "Land type mask" ;
		LANDTYPE72:units = "1" ;
		LANDTYPE72:_ChunkSizes = 1, 720, 1440 ;

// global attributes:
		:Title = "OLSON (2001) Land Map binary masks" ;
		:Conventions = "COARDS" ;
		:History = "Last modified on: Thu Jan  3 12:47:01 EST 2019" ;
		:ProductionDateTime = "Last modified on: Thu Jan  3 12:47:04 EST 2019" ;
		:ModificationDateTime = "Last modified on: Thu Jan  3 12:47:04 EST 2019" ;
		:format = "netCDF-3" ;
		:model = "GENERIC" ;
		:nlayers = "1" ;
		:start_date = "19850101" ;
		:start_time = "00:00:00.0" ;
		:end_date = "19860101" ;
		:end_time = "23:59:59.0" ;
		:delta_lon = "0.25" ;
		:delta_lat = "0.25" ;
		:delta_time = "000000" ;
}
```

Here is a plot of LANDTYPE30
![image](https://user-images.githubusercontent.com/12772306/135935891-89d8b881-9bb0-48fc-81c7-514195156f05.png)

</details>

**Comment by LiamBindle** [comment]
After reviewing this, I think we should change to the fractional masks (the `HcoDir/` version). I think that "conservatively regridding" the enums will miss classify the land type in mixed type areas. Thoughts? (pinging @lizziel @sdeastham)

**Comment by xpye98** [comment]
@LiamBindle Well, the `HcoDir/` version is not actually **fractional** , it's just **binary** (0 or 1) value (the `Interpolate`  is on when you used the Panoply, and that's confusing sometimes). 

So that's why I guess we select `ChemDir/` version for now to avoid some "middle" value between 0 and 1 after regridding in the case of `HcoDir/` version.

For mixed type areas problem, I think the model will first define how many types there are in a grid, and then do some weighting (at least for dry deposition).

**Comment by lizziel** [comment]
For the Olson land map containing integer values for all land types (variable OLSON) we use MAPL conservative fractional regridding. We specify which land type integer per target import in ExtData.rc, and MAPL essentially treats the file as binary values to get the fractions. For example, the entry for OLSON23 is [this](https://github.com/geoschem/geos-chem/blob/ba58da4a6e447e47714fe5e7da488d15432a6fdc/run/GCHP/ExtData.rc.templates/ExtData.rc.fullchem#L169):

`OLSON23 1 N F;23 - none none OLSON ./ChemDir/Olson_Land_Map_201203/Olson_2001_Land_Map.025x025.generic.nc`

The `F;23` tells MAPL to treat cells with value 23 as 1 and the rest as 0. Doing it this way allowed us to use the original land map file, although we did have to make some minor adjustments for compatibility with MAPL such as changing the land type from integer to float.

The file containing binary masks is for regridding using HEMCO since HEMCO does not have the fractional regridding capability that MAPL does. We created that file from the original. We could use it in GCHP as well, and change the regridding specification in ExtData.rc from fractional to just conservative, but it's not necessary.
